radio/src/gui/colorlcd/special_functions.cpp

```diff
@@ -362,7 +362,13 @@ class SpecialFunctionEditPage : public Page
     grid.spacer(PAGE_PADDING);
 
     CustomFunctionData *cfn = &functions[index];
-
+    
+    // custom label
+    new StaticText(window, grid.getLabelSlot(), "Custom Label", 0, COLOR_THEME_PRIMARY1);
```

Comment:
Please no fixed strings here, you have to add a new translation.

Reply:
@gagarinlg Done, though I had to use the defintion rather than the const char str, otherwise would not compile.
![image](https://user-images.githubusercontent.com/61736830/172303734-a351530c-5b97-4b26-ac73-71f124d07ddf.png)